### ISSUE TITLE
Problems with CSS for fieldsets

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#5053]
**Description of the bug**

In recent releases, I've noticed that fieldsets are displaying strangely on many of my sites (but not consistently). Has anyone else noticed this? 

I'm going to use this issue to track the various places I'm seeing the problems to see if it's caused by a recent CSS change in core, or something specific to one (or more?) of my sites. 

Collapsed fieldsets in a front-end theme as seen on [the DOCS site](https://docs.backdropcms.org/api/backdrop/core!modules!user!user.module/function/user_access/1):
<img width="843" alt="Screen Shot 2021-04-20 at 9 13 15 PM" src="https://user-images.githubusercontent.com/397895/115495758-390f7e00-a21d-11eb-86f2-cbff8c7ab040.png">

Collapsed fieldsets in the Seven theme (on a client site ivj) on the db log messages page:
<img width="1258" alt="Screen Shot 2021-04-20 at 9 13 43 PM" src="https://user-images.githubusercontent.com/397895/115495833-580e1000-a21d-11eb-9b12-4f07472bec98.png">

In all cases, adding `display: block;` to the fieldset fixes the problem. 

**Steps To Reproduce**

TBD

**Additional information**

Add any other information that could help, such as:
- Backdrop CMS version: 1.18.2 (docs site) 1.17.1 (client site: ivj)
- Browser(s) and their versions: Firefox 87.0 (chrome works fine)

**Comment by indigoxela** [comment]
The very first question, that comes to my mind: Browser(s) and their versions?

No, I didn't see this anywhere yet - neither in Firefox nor in Chrome.

**Comment by olafgrabienski** [comment]
Yes, the issue seems to be browser related. I've visited the [Docs site](https://docs.backdropcms.org/api/backdrop/core!modules!user!user.module/function/user_access/1) mentioned above on Mac OS X 10.15 and with three different browsers: Chrome, Safari, and Firefox.

While Chrome and Safari display the fieldsets fine, they look strange in Firefox (version: 87.0). Updating Firefox to version 88.0 fixed the issue for me.

**Comment by laryn** [comment]
Is the docs site on the latest version of Backdrop? 

I'm thinking of https://github.com/backdrop/backdrop/commit/c5fccff1e8b4c4d4f11a95eb7a7148fa4346d324

**Comment by jenlampton** [comment]
> Is the docs site on the latest version of Backdrop?

Yes, I think it is. But my client site is not. 

> The very first question, that comes to my mind: Browser(s) and their versions?

Both screenshots posted in the Original Post are from firefox, both those tow pages look fine in Chrome.

I've added the Backdrop and FireFox versions into the Original Post.

**Comment by laryn** [comment]
Is this to blame on the docs site? 
https://github.com/backdrop-ops/docs.backdropcms.org/blob/bf25ead4c7e9a171adc26c5aa0334f5f0f1f9c3e/www/themes/borg/css/system.theme.css#L442

**Comment by indigoxela** [comment]
> they look strange in Firefox (version: 87.0). Updating Firefox to version 88.0 fixed the issue for me.

Do we need a Browser quirk workaround for a version that's already outdated? If I get this right, it worked up to and including FF 86, is broken in 87 and is fixed in 88.

**Comment by jenlampton** [comment]
> Do we need a Browser quirk workaround for a version that's already outdated?

I don't think so. FF usually auto-updates on restart so it shouldn't bother people for that long.

> Updating Firefox to version 88.0 fixed the issue for me.

I quit and restarted, FF updated to 88.0, and it looks fine for me now in FF. Should we close this issue?

**Comment by BWPanda** [comment]
Duplicate of https://github.com/backdrop-ops/docs.backdropcms.org/issues/130
